boms/user/server/ee/ee-preview/ee-preview-with-tools/pom.xml

```diff
@@ -0,0 +1,168 @@
+        <artifactId>wildfly-server-ee-preview-builder</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>33.0.0.Beta1-SNAPSHOT</version>
+    </parent>
+
+    <packaging>pom</packaging>
+
+    <artifactId>wildfly-server-ee-preview-with-tools-builder</artifactId>
+
+    <name>WildFly BOM Builders: Server EE Preview with Tools Builder</name>
+    <description>This artifact builds a bill of materials (BOM) for WildFly Server EE Preview with Deployment and Testing Tools</description>
+
+    <dependencyManagement>
+        <dependencies>
+
+            <!-- imports the related server's test BOM -->
+            <dependency>
+                <groupId>org.wildfly</groupId>
+                <artifactId>wildfly-standard-test-bom</artifactId>
+                <version>${ee.maven.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <!-- The WildFly plugin deploys your war to a local WildFly container -->
+                <!-- To use, set the JBOSS_HOME environment variable and
+                    run: mvn package wildfly:deploy -->
+                <plugin>
+                    <groupId>org.wildfly.plugins</groupId>
+                    <artifactId>wildfly-maven-plugin</artifactId>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-bom-builder-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>build-bom</id>
+                        <goals>
+                            <goal>build-bom</goal>
+                        </goals>
+                        <configuration>
+                            <parent>
+                                <groupId>${bomGroupId}</groupId>
+                                <artifactId>${bomArtifactIdPrefix}-ee-preview</artifactId>
+                                <version>${project.version}</version>
+                                <relativePath/>
+                            </parent>
+                            <bomGroupId>${bomGroupId}</bomGroupId>
+                            <bomArtifactId>${bomArtifactIdPrefix}-ee-preview-with-tools</bomArtifactId>
+                            <bomVersion>${project.version}</bomVersion>
+                            <bomName>${full.dist.product.release.name} BOMs: EE Preview with Tools</bomName>
+                            <bomDescription>Dependency Management for ${full.dist.product.release.name} EE Preview, with Deployment and Testing Tools.</bomDescription>
```

Comment:
'EE' doesn't belong in this text or in line 88.

WildFly Preview is a general purpose preview, not just for new things in Jakarta EE.